pinot-transport/src/main/java/com/linkedin/pinot/routing/builder/KafkaLowLevelConsumerRoutingTableBuilder.java

```diff
@@ -55,6 +55,7 @@ public void init(Configuration configuration) {
       final String targetServerCountPerQuery = configuration.getString("realtimeTargetServerCountPerQuery");
       try {
         TARGET_SERVER_COUNT_PER_QUERY = Integer.parseInt(targetServerCountPerQuery);
+        LOGGER.info("Using target server count of {}", TARGET_SERVER_COUNT_PER_QUERY);
```

Comment:
can u add in the log that this is for realtime nservers? A more descriptive log may also help.